gctscript/wrappers/validator/validator.go

```diff
@@ -212,3 +222,41 @@ func (w Wrapper) WithdrawalFiatFunds(exch, _ string, _ *withdraw.Request) (out s
 
 	return "123", nil
 }
+
+// OHLCV returns open high low close volume candles for requested exchange/pair/asset/start & end time
+func (w Wrapper) OHLCV(exch string, p currency.Pair, a asset.Item, start, end time.Time, i time.Duration) (kline.Item, error) {
+	if exch == exchError.String() {
+		return kline.Item{}, errTestFailed
+	}
+	var candles []kline.Candle
+
+	candles = append(candles, kline.Candle{
+		Time:   start,
+		Open:   validatorOpen,
+		High:   validatorHigh,
+		Low:    validatorLow,
+		Close:  validatorClose,
+		Volume: validatorVol,
+	})
+
+	for x := 0; x < 200; x++ {
+		r := validatorLow + rand.Float64()*(validatorHigh-validatorLow)
+		candle := kline.Candle{
+			Time:   start.Add(i),
```

Comment:
This seems like its going to be the same time for every candle.

Reply:
its just used for testing and when uploading the timestamp isn't important 

for example in unit test i just call it as 
```
_, err := testWrapper.OHLCV("test", c, asset.Spot, time.Now().Add(-24*time.Hour), time.Now(), kline.OneDay)
```

yeah everything after the first day will return the same timestamp but its mainly just to satisfy validation on uploading that you are uploading a script that will compile and run

Reply:
👍 cool!

Reply:
Still tACK

Reply:
if you think there is ever a case where we might use it to generate realistic fake data i am happy to expand it 

Just wanted a way to pass somewhat usable data back to indicators during unit vests/validation

Reply:
When we need it we can upgrade then I guess. As long as its there. @thrasher- Thoughts?

Reply:
We'll be creating simulations for bull markets, bear markets, parabolic advances and also savage declines to test how algos perform (esp for market neutral ones), but this is out of scope for this PR so this is fine as is

Reply:
I went ahead and pushed changes to flesh out the data and generate random times based on the interval as it was not a huge change


Also please feel free to request any other indicators you find useful
I am adding Bollinger Bands linking in now